packages/opentelemetry-plugin-grpc/package.json

```diff
@@ -40,14 +40,17 @@
   "devDependencies": {
     "@types/mocha": "^5.2.7",
     "@types/node": "^12.6.9",
+    "@types/protobufjs": "^6.0.0",
```

Comment:
This package has been deprecated, why do we need this -  https://www.npmjs.com/package/@types/protobufjs?

Reply:
Oops not sure how this snuck it -- removed it